src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -65,8 +65,8 @@
   </StringProperty>
   <StringProperty Name="DefineConstants" DisplayName="Define Constants" Visible="True"/>
   <EnumProperty Name="PlatformTarget" DisplayName="Platform Target" Visible="True"/>
-  <StringProperty Name="Prefer32Bit" DisplayName="Prefer 32Bit" Visible="True"/>
-  <StringProperty Name="AllowUnsafeBlocks"  Default="False"  DisplayName="Allow unsafe code" Visible="True"/>
+  <BoolProperty Name="Prefer32Bit" DisplayName="Prefer 32Bit" Visible="True"/>
```

Comment:
Death by a thousand cuts - there's a bunch of things property page related that are not quite right. In this specific case those values should have been Boolean not strings. That fixes the dropdown correctly in the property grid. This PR has 4 changes, they're split up into different commits as each one addresses a different thing.